pkg/rke2/config.go

```diff
@@ -319,16 +319,13 @@ func newRKE2ServerConfig(opts ServerConfigOpts) (*ServerConfig, []bootstrapv1.Fi
 			})
 		}
 
-		rke2ServerConfig.EtcdSnapshotDir = opts.ServerConfig.Etcd.BackupConfig.S3.Folder
```

Comment:
will only removing this line solve the issue? or as @Danil-Grigorev says about other values, it can be populated anyway